### ISSUE TITLE
No padding in issue preview on hover

### ISSUE THREAD

**Issue by stuzer05** [go-gitea/gitea#25249]
### Description

When you hover over an issue number, popup has no paddings

https://try.gitea.io/stuzer05/test-preview/pulls/1

### Gitea Version

trunk

### Can you reproduce the bug on the Gitea demo site?

Yes

### Screenshots

![image](https://github.com/go-gitea/gitea/assets/23124682/295df8aa-09f1-4e1e-bece-a06b89e7f3ac)

### How are you running Gitea?

https://try.gitea.io

### Database

None

**Comment by wxiaoguang** [comment]
It seems to be caused by 

```
.tippy-box[data-theme="menu"] .tippy-content {
  padding: 0;
}
```

@silverwind I think it's a regression of making the theme=role for menus?

**Comment by silverwind** [comment]
Yes, it's a regression. The `code-line-menu` adds it's own padding, so this tippy content needs to set a padding itself.

**Comment by wxiaoguang** [comment]
The default "theme" by "role" still seems a little confusing to me.

I tried to propose  Fix issue popup padding #25254  , use "default" for the theme (the "default" is a non-existing selector, so I think it does it work). Does it look good to you or do you have better approaches?
